app/scripts/controllers/chapter.js

```diff
@@ -71,13 +71,13 @@ angular.module('gdgxHubApp')
     });
 
     $http.get('https://www.googleapis.com/plus/v1/people/' + $routeParams.chapterId +
-    '?fields=aboutMe,image&key=AIzaSyD7v04m_bTu-rcWtuaN3fTP9NBmjhB7lXg')
+    '?fields=aboutMe,image&key=' + config.GOOGLE_API_KEY)
```

Comment:
I am also thinking to do a normal GET operations for Google Plus access in Frisbee. We are using Google Plus Api Client for java which has dependency to Gson, Apache Http Client etc.